### PR TITLE
{chem}[foss/2016b] libxc v3.0.1

### PR DESCRIPTION
**Pull request by pouillon**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/l/libxc/libxc-3.0.1-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+source_urls = ['http://www.tddft.org/programs/octopus/down.php?file=libxc/3.0.1/']
+
+configopts = 'FC="$F77" FCFLAGS="$FFLAGS" --enable-shared --enable-fortran'
+
+# From the libxc mailing list
+# To summarize: expect less tests to fail in libxc 2.0.2, but don't expect
+# a fully working testsuite soon (unless someone wants to volunteer to do
+# it, of course  ) In the meantime, unless the majority of the tests
+# fail, your build should be fine.
+# runtest = 'check'
```
Comment (boegel) [inline easybuild/easyconfigs/l/libxc/libxc-3.0.1-foss-2016b.eb]: @pouillon We recently have consistently specified `toolchainopts` to fix broken tests, and also enabled the tests, for recent `libxc` easyconfigs, see for example #5256.

Can you sync this `libxc` (and the one in your other `libxc` PRs #5398, #5400, #5401) with the recent `libxc` easyconfigs in https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/l/libxc?

Comment (pouillon) [reply easybuild/easyconfigs/l/libxc/libxc-3.0.1-foss-2016b.eb]: Thanks for the quick reply. I hadn't dared enabling the tests, although they succeed on my machine. I looked both at #5256 and the HEAD of easyconfigs/develop to synchronize my files. It's a nice improvement over my previous versions.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2113.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2876f87cc3b51b11e0ffe6393bb74195 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @pouillon!

**Pull request merged**